# neo4j-examples

fixed the yelp-data-challenge import problem: 
<pre>
Traceback (most recent call last):
File "json_to_csv.py", line 109, in
for category in item["categories"]:
TypeError: 'NoneType' object is not iterable
</pre?
